ui/design-system-core/src/components/ErrorLoader/error-card.tsx

```diff
@@ -26,27 +28,30 @@ const ErrorRenderer: React.FC<PropsWithChildren<ErrorRendererProps>> = props =>
 };
 
 const ErrorCard: React.FC<PropsWithChildren<ErrorLoaderProps & { imageSrc: string }>> = props => {
-  const { children, title, imageSrc, details, devDetails, className } = props;
+  const { children, title, imageSrc, details, devDetails, style } = props;
 
   const isDevMode = false;
   const message = details ?? devDetails;
 
   return (
-    <div className={tw(`flex flex-col items-center p-[1em] bg-white ${className}`)}>
+    <div className={tw(apply`flex flex-col items-center p-[1em] bg-white ${style}`)}>
```

Comment:
no need for apply, you can use [tx utility](https://twind.style/reference#utilities)